src/transformers/utils/notebook.py

```diff
@@ -318,13 +319,16 @@ def on_evaluate(self, args, state, control, metrics=None, **kwargs):
                 values["Epoch"] = int(state.epoch)
             else:
                 values["Step"] = state.global_step
-            values["Validation Loss"] = metrics["eval_loss"]
```

Comment:
this line seemed to be redundant since we later loop over `metrics.items()`, so i removed it